libs/components/forms/src/lib/modules/input-box/input-box.component.spec.ts

```diff
@@ -802,6 +802,32 @@ describe('Input box component', () => {
       );
     });
 
+    it('should hide hint text when `hideHintText` is called with `true`', () => {
+      const fixture = TestBed.createComponent(InputBoxFixtureComponent);
+      fixture.detectChanges();
+      fixture.componentInstance.inputBoxHostServiceComponent?.hideHintText(
+        true,
+      );
+      fixture.detectChanges();
+
+      const els = getDefaultEls(fixture, 'input-host-service');
+
+      expect(els.hintTextEl).not.toBeVisible({ checkCssVisibility: true });
```

Comment:
I added the following to this test and it failed. I imagine you're hiding the text (instead of clearing it) so that the form heights don't "jump" when selecting a country. I noticed that the country input still has placeholder text; instead of hiding the hint text, could we move the "Search for a country" placeholder to the hint text when the country selector is shown? That way, the field height won't change between states.
```
await expectAsync(fixture.nativeElement).toBeAccessible();
```

Reply:
No. We had a discussion in flex and this approach is what was decided on. Matt very strongly did not want to move the country search placeholder to the hint text at this time and so he, Paul, and I agreed to go with the visibility approach. 

I’ll look into the accessibility issue though.